src/vs/workbench/contrib/debug/browser/runAndDebugAccessibilityHelp.ts

```diff
@@ -7,18 +7,20 @@
 import { ServicesAccessor } from 'vs/editor/browser/editorExtensions';
 import { AccessibleViewProviderId, AccessibleViewType, IAccessibleViewContentProvider } from 'vs/platform/accessibility/browser/accessibleView';
 import { IAccessibleViewImplentation } from 'vs/platform/accessibility/browser/accessibleViewRegistry';
-import { ContextKeyExpr } from 'vs/platform/contextkey/common/contextkey';
 import { Disposable } from 'vs/base/common/lifecycle';
 import { AccessibilityVerbositySettingId } from 'vs/workbench/contrib/accessibility/browser/accessibilityConfiguration';
 import { localize } from 'vs/nls';
 import { ICommandService } from 'vs/platform/commands/common/commands';
 import { IViewsService } from 'vs/workbench/services/views/common/viewsService';
 import { AccessibilityHelpNLS } from 'vs/editor/common/standaloneStrings';
+import { CONTEXT_DEBUG_VIEW_TOOLBAR_FOCUSED, CONTEXT_DEBUG_VIEW_FOCUSED } from 'vs/workbench/contrib/debug/common/debug';
+import { WelcomeView } from 'vs/workbench/contrib/debug/browser/welcomeView';
+import { ContextKeyExpr } from 'vs/platform/contextkey/common/contextkey';
 
 export class RunAndDebugAccessibilityHelp implements IAccessibleViewImplentation {
 	priority = 120;
 	name = 'runAndDebugHelp';
-	when = ContextKeyExpr.equals('activeViewlet', 'workbench.view.debug');
```

Comment:
I wonder if you could combine `activeViewlet` with a key that already exists like `sideBarFocus`? If that doesn't work, could we add another key that covers all cases rather than having to add an individual key for each individual component that we might have to track focus on? I assume you'd have to do this for everything that the accessibility view can be activated on

Reply:
Or instead of having individual commands controlled by context keys, you could have one command that looks up the right context dynamically.

Reply:
Not sure if either of those is possible, but seems like something that can be deferred so this bug is fixed?

Reply:
Can't the debug view be moved elsewhere ie not just in the sidebar?

Reply:
And sure, we could add a key that covers these two and any other cases, but I'm not sure that's better than just using a few specific ones

Reply:
> Can't the debug view be moved elsewhere ie not just in the sidebar?

Ah, true 😅. But that makes me realize, what do you want to happen when I move the variables view out of the debug viewlet? Should I still get debug accessibility help for it? What if the terminal is in the debug viewlet? I wonder if you really want to capture the individual views.

I think that adding a context key for one particular menu button is just very very granular. We have dozens of buttons, and if you really want to capture whether focus is inside the debug view container, why not have a context key for it? Looks like we have a `focusedView` and I think `focusedViewContainer` makes sense. It would probably be easier to implement properly. Then we get it for all view containers too.

Reply:
That seems outside of the scope of this issue

Reply:
Also want to keep this fix simple
Since it's a candidate